library/src/main/java/pl/rspective/voucherify/android/view/VoucherCheckoutView.java

```diff
@@ -28,23 +35,34 @@
 
     private VoucherifyAndroidClient voucherifyClient;
     private OnValidatedListener onValidatedListener;
+    private TextInputLayout voucherCodeLabel;
```

Comment:
I think that we could add option to provide 'error' message but we will think about this later. Just want to point it out